styleguide/content/brand-identity/colors.md

```diff
@@ -0,0 +1,157 @@
+
+### Transparency
+
+We manage a very small set of “base” colors, with related colors being transparencies of the parent. We represent the “base” color with a value of 100 (eg. ‘Red 100’), and transparencies of that color are named by the percentage of opacity (eg. ‘Red 54’ is equivalent to Red 100 with a 54% opacity). We use this type of palette so that we are technically only managing a small set of colors and can update them easier if needed. If there are ever any challenges or specific issues related to using a transparent color in practice, please inform your team’s designer or Web Infrastructure.
+
+## Usage
+
+```css
+@import '@mavenlink/design-system/src/styles/colors-v2.css';
+
+.example {
+    background-color: var(--mds-blue-54);
+}
+```
+
+<style>
+  .color-container {
+    display: flex;
+    flex-direction: column;
+    margin-bottom: var(--spacing-x-large);
+    margin-left: var(--spacing-small);
+    margin-right: var(--spacing-small);
+    max-height: 300px;
+    max-width: 300px;
+    width: 300px;
+  }
+  
+  .dark-contrast {
+    color: var(--white);
+    padding: var(--spacing-medium);
+    font-family: var(--mavenlink-type-font-family);
+  }
+  
+  .light-contrast {
+    color: var(--black);
+    padding: var(--spacing-medium);
+    font-family: var(--mavenlink-type-font-family);
+  }
+  
+  .other-colors {
+    display: flex;
+    flex-wrap: wrap;
+  }
+</style>
+
+<h3>Brand Colors</h3>
+<div class="color-container">
+  <div class="dark-contrast" style="background-color: var(--mds-brand-text);">Brand Text</div>
+  <div class="dark-contrast" style="background-color: var(--mds-brand-100);">Brand 100</div>
+  <div class="light-contrast" style="background-color: var(--mds-brand-54);">Brand 54</div>
+  <div class="light-contrast" style="background-color: var(--mds-brand-38);">Brand 38</div>
+  <div class="light-contrast" style="background-color: var(--mds-brand-12);">Brand 12</div>
+</div>
+
+<h3>Base Colors</h3>
+<div class="color-container">
+  <div class="light-contrast" style="background-color: var(--white)">White</div>
+  <div class="dark-contrast" style="background-color: var(--black)">Black</div>
+  <div class="dark-contrast" style="background-color: var(--mds-grey-87);">Grey 87</div>
+  <div class="light-contrast" style="background-color: var(--mds-grey-54);">Grey 54</div>
+  <div class="light-contrast" style="background-color: var(--mds-grey-38);">Grey 38</div>
+  <div class="light-contrast" style="background-color: var(--mds-grey-12);">Grey 12</div>
+  <div class="light-contrast" style="background-color: var(--mds-grey-8);">Grey 8</div>
+  <div class="light-contrast" style="background-color: var(--mds-grey-3);">Grey 3</div>
+</div>
+
+<h3>Other Colors</h3>
+<span class="other-colors">
```

Comment:
This was a workaround due to a bug, found [here](https://github.com/styleguidist/react-styleguidist/issues/1589)